portfolio/src/main/java/com/google/sps/servlets/Comment.java

```diff
@@ -0,0 +1,12 @@
+package com.google.sps.servlets;
```

Comment:
I think many of these changes were first made in the previous PR.  Could you try going to the page https://github.com/olu144/step/pull/10 and clicking "Resolve conflicts".  and following the steps to pick which code you want included in your PR?  Right now the file diff is still showing code you already merged in a previous PR, but I think once you do the "Resolve conflicts" it should only show the new code

Reply:
Addressed in latest commit.